Anytype/Sources/PresentationLayer/Modules/SpaceSwitch/SpaceSwitchView.swift

```diff
@@ -11,7 +11,6 @@ struct SpaceSwitchView: View {
     @StateObject private var model: SpaceSwitchViewModel
     @Environment(\.dismiss) private var dismiss
     
-    @State private var headerSize: CGSize = .zero
```

Comment:
onSettingsSelected needed in SpaceSwitchModuleOutput?

Reply:
I decided to keep it for a while just in case product team will decide to revert this change